src/lib/pixel-select-drop-down/pixel-select-drop-down.tsx

```diff
@@ -211,6 +218,47 @@ export const PixelDropDown = React.forwardRef<HTMLDivElement, DropDownProps>(
     const truncate = (label, max) => {
       return label?.length > max ? label.substr(0, max - 1) + '...' : label
     }
+
+    const [selectedOption, setSelectedOption] = React.useState([])
+
+    const handleRadioClick = (option) => {
+      rest.onChange &&
+        rest.onChange({
+          target: {
+            value: option.value,
+            options: [{ text: option.label }],
+            selectedIndex: 0
+          }
+        })
+      setIsOptionsOpen(isShowRadio ? true : false)
+      setDropdownPosition({
+        ...dropdownPosition,
+        left: ''
+      })
+      setSelectedOption(option.value)
+    }
+
+    const handleCheckboxClick = (option) => {
+      const isSelected = selectedOption.includes(option)
+
+      if (isSelected) {
+        setSelectedOption(selectedOption.filter((item) => item !== option))
+      } else {
+        setSelectedOption([...selectedOption, option])
+      }
+
```

Comment:
@masabharry you are updating checkbox values locally here what if values in not updated somehow from where we use them? 
same in the radio case.

Reply:
Sorry, but Saifi Bhai I can't understand this question.